crypto/context.c

```diff
@@ -192,21 +183,15 @@ void *openssl_ctx_get_data(OPENSSL_CTX *ctx, int index,
     if (ctx == NULL)
```

Comment:
so, when I look at the line above `ctx = openssl_ctx_get_concrete(ctx);`
it seems likely that many threads can call `openssl_ctx_get_data(NULL,...)`
and all of them will use `ctx == default_context` (except in FIPS mode) right?